etc/profile-m-z/menulibre.profile

```diff
@@ -0,0 +1,53 @@
+# Firejail profile for menulibre
+# Description: Create desktop and menu launchers easily
+# This file is overwritten after every install/update
+# Persistent local customizations
+include menulibre.local
+# Persistent global definitions
+include globals.local
+
+ignore read-only ${HOME}/.local/share/applications
+ignore read-only ${HOME}/.config/menus
+
+noblacklist ${HOME}/.gnome/apps
+
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc 
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+include disable-xdg.inc
+
+include whitelist-runuser-common.inc 
```

Comment:
wusc?

whitelist?

Reply:
Half of the list is not being populated with desktop entries, tried adding `whitelist /usr/share/applications`
`whitelist /usr/share/menulibre` with wusc.

Reply:
Works the following? If not try with `--build`. If thats still don't work, leave it.
```
whitelist /usr/share/applications
whitelist /usr/share/menulibre
whitelist /usr/share/desktop-directories
include wusc.inc
```

Reply:
Works,thanks but some of the entries have no icon picture. Tried adding `~./local/share/icons,/usr/share/icons,pixmaps,mime` etc. Build doesn't give anything more than this.

Reply:
flatpak: `/var/lib/flatpak/exports/share/{applications,icons}` and `~/.local/share/flatpak/exports/share/{applications,icons}`

> some of the entries have no icon picture.

Are these icons specified by name or by path?

Reply:
Some them name; by custom path will be broken for some. Thanks for snap/flatpak.

Reply:
Can you `locate` for the name?

Reply:
Found them.. Most of them are in, `/var/lib/app-info/icons/. Does this vary by distro?

Reply:
Fedora seems to use `/usr/share/app-info/`.

Reply:
Arch uses `/usr/share/icons/` in `hicolor`

Reply:
FWIW:
- Snap: don't use bubblewrap
- flatpak: relies on bubblewrap
- nautilus: runs thumbnailers in a sandbox with a read-only filesystem

Reply:
> flatpak: relies on bubblewrap

Yeah but not atril, liferea,software-properties-common/-gtk,update-manager. etc. These are from Ubuntu's own repo and shouldn't require a sandbox if there's no flatpak. [Idk what they are doing.](https://www.bleepingcomputer.com/news/security/ubuntu-is-undoing-a-gnome-security-feature/)